tests/components/airly/test_init.py

```diff
@@ -124,17 +126,23 @@ async def test_update_interval(hass, aioclient_mock):
         assert len(hass.config_entries.async_entries(DOMAIN)) == 1
         assert entry.state == ENTRY_STATE_LOADED
 
-        # We have 160 minutes to midnight, 15 remaining requests and one instance so
-        # next update will be after ceil(160 / 15 * 1) = 11 minutes
-        future = POINT + timedelta(minutes=11)
+    # We have 160 minutes to midnight, 15 remaining requests and one instance so next
+    # update will be after ceil(160 / 15 * 1) = 11 minutes
+    future = POINT + timedelta(minutes=11, seconds=1)
+    with patch(
+        "homeassistant.components.airly.dt_util.utcnow", return_value=future
```

Comment:
If we store the patch context manager mocks in variables we don't need to create new contexts when changing the return value of the mocks. We can just set a new return value on the mocks.

```py
with patch(...) as mock:
    mock.return_value = value_1
    ...
    mock return_value = value_2
```